modules/apps/29-fee/keeper/msg_server_test.go

```diff
@@ -106,6 +108,15 @@ func (suite *KeeperTestSuite) TestPayPacketFee() {
 			},
 			true,
 		},
+		{
+			"refund account is module account",
```

Comment:
looking at it now, it might make more sense to put this in the escrow test suite

Reply:
it'd be nice if there was a test for distributing to a module acc